src/components/Callout/Callout.md

```diff
@@ -0,0 +1,21 @@
+### Notes for use
+
+Callout is simply exported from Fabric, with some minor YamUI styling. It is meant to be used as a utility component by other YamUI components like Hovercard and Tooltip, you probably don't need to use it directly in your application UIs.
+https://developer.microsoft.com/en-us/fabric#/components/callout
```

Comment:
I'd wrap the URL in `<>`, or turn some of the words in the paragraph into a link with `[]`. GitHub is doing its own parsing of URLs and turning them into links, but it's not standard Markdown behavior.

Reply:
done